teachers_digital_platform/jinja2/teachers_digital_platform/crt-start.html

```diff
@@ -90,7 +90,7 @@ <h2 class="h4">Not sure how to get ready for your curriculum review?</h2>
             <a class="a-link a-link__icon-after-text"
                 target="_blank"
                 rel="noopener noreferrer"
-                href="https://files.consumerfinance.gov/f/documents/cfpb_curriculum-review-tool_checklist-to-get-started_checklist.pdf">
+                href="https://files.consumerfinance.gov/f/documents/cfpb_youth-financial-education-curriculum-review-tool.pdf">
                     <span class="a-link_text">Download our curriculum review checklist</span>
```

Comment:
Are you sure this is correct? This link (and the header introducing it) seem to indicate that it's specifically for the get-started checklist that is originally linked.

Reply:
Good catch! I changed the wrong URL